kerl

```diff
@@ -782,8 +782,16 @@ maybe_patch_catalina() {
     fi
 }
 
+is_osx() {
+    uname="$(uname -r | grep "^$1.*$")"
+    if [ "$uname" != "" ]; then
+        return
+    fi
+    false
```

Comment:
```suggestion
is_osx() {
    uname -r | \grep "^$1.*$" > /dev/null
```

I would have written it like this. It is the form used in many other places in the `kerl` code.

Reply:
Cool. I can change that. It's more compact. Thanks. @jadeallenx, are you Ok with this suggestion?

Reply:
Yes, definitely prefer that, thanks

Reply:
Done. Thanks, @garazdawi, I just had to press the Commit suggestion button. 😄